docs/en_US/deployment/on-gcp-gke.md

```diff
@@ -1,60 +1,160 @@
-# Deploy EMQX on Google Cloud GKE
+# Deploy EMQX on Google Kubernetes Engine
 
-Google Kubernetes Engine (GKE) provides a managed environment for deploying, managing, and scaling your containerized applications using Google infrastructure. The GKE environment consists of multiple machines (specifically, Compute Engine instances) grouped together to form a cluster.
+## Overview
 
-## Create GKE Cluster
+This guide will walk you through the process of deploying EMQ X, an open-source MQTT broker, on Google Kubernetes Engine (GKE). By following these steps, you'll learn how to create and configure an EMQ X deployment on GKE.
 
-Log in Google Cloud GKE console and enter the page for creating GKE Cluster. For details: [Create GKE cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-an-autopilot-cluster)
+## Prerequisites
 
-## Access Kubernetes Cluster
++ A Google Kubernetes Engine (GKE) cluster, for more information, see [Creating an autopilot cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-an-autopilot-cluster)
++ MQTTX CLI, A user-friendly MQTT 5.0 command line tool, download it [here](https://mqttx.app/cli)
 
-For details: [kubeconfig](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl)
 
-## Configure Cert Manager
+## Deploy EMQX on GKE
 
-To install `cert-manager` read the official docs:
+### Deploying EMQX Operator
+
+**Deploying Cert Manager**
+
+To install `cert-manager`, consult the official documentation:
 
 - [GKE Autopilot](https://cert-manager.io/docs/installation/compatibility/#gke-autopilot)
 - [Private GKE Cluster](https://cert-manager.io/docs/installation/compatibility/#gke)
 
-Don't forget to install CRDs when running `helm` using `--set installCRDs=true`.
+Remember to install CRDs when running `helm` with the `--set installCRDs=true` flag.
 
-> More info at [cert-manager](https://cert-manager.io).
+> More information can be found at [cert-manager](https://cert-manager.io).
 
-## EMQX Operator
+To install `emqx-operator`, refer to the official [docs](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/getting-started/getting-started.md)
```

Comment:
To install `emqx-operator`, refer to the official [docs](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/getting-started/getting-started.md)

::: tip

The EMQX Operator relies on `cert-manager`,  to install `cert-manager`, consult the official documentation:

- [GKE Autopilot](https://cert-manager.io/docs/installation/compatibility/#gke-autopilot)
- [Private GKE Cluster](https://cert-manager.io/docs/installation/compatibility/#gke)

Remember to install CRDs when running `helm` with the `--set installCRDs=true` flag.
:::